RulesCommon.py

```diff
@@ -0,0 +1 @@
+allowed_globals = {}
```

Comment:
Since this file is added anyway, maybe we could move the repeated escape_name code in here?